modules/update-manifest-application.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-manifest-application_{context}"]
+= Understanding how manifests are applied during an update
+
+Some manifests supplied in a release image need to be applied in a certain order because of the dependencies between them.
+For example, the `CustomResourceDefinition` resource must be created before the matching custom resources.
+Additionally, there is a logical order in which the individual Cluster Operators need to be updated to minimize disruption in the cluster.
+The Cluster Version Operator (CVO) implements this logical order through the concept of Runlevels.
+
+These dependencies are encoded in the filenames of the manifests in the release image:
+
+[source, terminal]
+----
+0000_<runlevel>_<component>_<manifest-name>.yaml
+----
+
+For example:
+
+[source, terminal]
+----
+0000_03_config-operator_01_proxy.crd.yaml
+----
+
+The CVO internally builds a dependency graph for the manifests, obeying the following rules:
+
+* During an update, manifests at a lower Runlevel will be applied before those at a higher Runlevel.
+
+* Within one Runlevel, manifests for different components may be applied in parallel.
+
+* Within one Runlevel, manifests for a single component will be applied in lexicographic order.
+
+The CVO then applies manifests following the generated dependency graph.
+
+For some resource types, the CVO monitors the resource after its manifest is applied, and only considers it to be successfully updated after the resource reaches a stable state.
+Achieving this state can take some time.
+This is especially true for Cluster Operators, which may perform their own update actions in the cluster after the CVO deploys their new versions.
+While the additional update actions take place, these Cluster Operators will temporarily set their `Progressing` condition to `True`.
+
+The CVO waits until all Cluster Operators in the Runlevel:
+
+* Are Available=True,
+
+* Are Degraded=False, and
+
+* Declare they have achieved the desired version in their ClusterOperator resource
+
+before it proceeds to the following Runlevel.
+Some actions can take significant time to finish, but the CVO needs to wait on their completion to ensure the following Runlevels can proceed safely.
```

Comment:
_the following Runlevels_ suggests to me that you are listing the Runlevels. 
What actions do you mean?
I'm not sure this is a _but_ situation (seems a bit negative).  Maybe two sentences?

```suggestion
Some actions can take significant time to finish. The CVO waits for the actions to complete in order to ensure the next Runlevel can proceed safely.
```

Reply:
I switched "following" with "subsequent" to avoid this confusion, let me know if that doesn't work for you